tools/shared.py

```diff
@@ -1627,7 +1627,7 @@ def llvm_as(input_filename, output_filename=None):
   nm_cache = {} # cache results of nm - it can be slow to run
 
   @staticmethod
-  def llvm_nm(filename, stdout=PIPE, stderr=None):
+  def llvm_nm(filename, stdout=PIPE, stderr=None, **kwargs):
```

Comment:
Yeah, something like `include_internal=False` is what I meant. It's still a kwarg, it just doesn't use the `**kwargs` (I guess you'd call that "dynamic kwargs"?) Python feature.